src/button/index.tsx

```diff
@@ -41,10 +41,7 @@ const StyledButton = styled(Button)`
   }
 
   &:hover,
-  &:focus,
```

Comment:
Focus is also call onmousdown depending on browsers, which might results in a quick flash of focus state when clicking on button. So maybe it should apply only on keyboard navigation like we did in Kairos using focus-visible. 
https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible
wdyt?

Reply:
✅ PTAL

Reply:
This won't work, because it means that we need to pass a `.focus-visible` class to each component.
We have this in Kairos:
```
/* Remove default focus outline when focus is not visible. */
body:not(.focus-visible) :focus {
  outline: none;
}
```
So depending on CSS specificity your previous commit might work sorry 🤦‍♂